src/main/java/cinema/model/User.java

```diff
@@ -0,0 +1,206 @@
+package cinema.model;
+
+import java.util.Objects;
+import java.util.Optional;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long userId;
+    private String firstName;
+    private String lastName;
+    private int age;
+    @Column(unique = true)
+    private String login;
+    @Column(unique = true)
+    private String email;
+    private Integer phone;
+    private String password;
+    private byte[] salt;
+
+    public User() {
+    }
+
+    public User(String firstName, String lastName,
+                int age, String login, String email, String password) {
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.age = age;
+        this.login = login;
+        this.email = email;
+        this.password = password;
+    }
+
+    public User(String firstName, String lastName,
+                int age, String login, String email, String password, byte[] salt) {
+        this.firstName = firstName;
```

Comment:
@Merkald Maybe better to use Builder pattern? This constructor looks over complicated. Or even if you want to use constructors - can we use overloading here?